scripts/vendor-boringssl.sh

```diff
@@ -264,6 +264,7 @@ echo "RENAMING header files"
 echo "PATCHING BoringSSL"
 git apply "${HERE}/scripts/patch-1-inttypes.patch"
 git apply "${HERE}/scripts/patch-2-arm-arch.patch"
+git apply "${HERE}/scripts/patch-3-weak-linking.patch"
```

Comment:
@Lukasa is this something that should redone by BoringSSL itself /cc @davidben ?

Reply:
Unclear: see the conversation in #189.